src/mongo/db/curop.h

```diff
@@ -305,6 +306,10 @@ class CurOp {
         if (_dbprofile <= 0)
             return false;
 
+        if (serverGlobalParams.rateLimit > 1 && _dbprofile >= 2 && ms < serverGlobalParams.slowMS) {
```

Comment:
Can you explain how you limit events for the case of `profile = 1` and `rateLimit > 1`?